### PR TITLE
Exercise5

### PR DESCRIPTION
**Pull request by metis-macys-66898**

### CODE REVIEW

File: curriculum/js1/4.js [hunk 1]
```diff
@@ -6,8 +6,15 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (x, string = "") => {
+   if (1 < x) {
+       x -= 1;
+       console.log(string);
```
Comment (songz) [inline curriculum/js1/4.js]: remove console.logs

File: curriculum/js1/4.js [hunk 2]
```diff
@@ -6,8 +6,15 @@
...
+       return solution(x, string) + string;
+   } else {
```
Comment (songz) [inline curriculum/js1/4.js]: since you are returning in your if statement, you don't need an else here.

File: curriculum/js1/4.js (hunk 1 above)
Comment (metis-macys-66898) [reply curriculum/js1/4.js]: I just removed my else case.

**Comment by songz** [comment]
Closing because you should be sending MRs to your own master.

**Pull request closed without merging**